### ISSUE TITLE
can you provide finetuning code? error

### ISSUE THREAD

**Issue by zdlpsina** [deepopinion/domain-adapted-atsc#5]
the original finetune_on_pregenerated.py from hugging face/pytorch-transformers repository actually has some bugs:
        optimizer = AdamW(optimizer_grouped_parameters,
                             lr=args.learning_rate,
                             warmup=args.warmup_proportion,
                             t_total=num_train_optimization_steps)
actually "warmup,t_total" shouldn't be used.

Besides, I first run "Pregenerating training data", then the "Training on pregenerated data" as the readme from "huggingface," said. but the final files didn't improve the performance

from the code in the original finetune_on_pregenerated.py：
"      output_model_file = os.path.join(args.output_dir, WEIGHTS_NAME)
    output_config_file = os.path.join(args.output_dir, CONFIG_NAME)

    torch.save(model_to_save.state_dict(), output_model_file)
    model_to_save.config.to_json_file(output_config_file)
    tokenizer.save_vocabulary(args.output_dir)
"
the final finetuning model files are different from your provides;
![1](https://user-images.githubusercontent.com/37832104/66387930-a0792a80-e9f7-11e9-8ca3-8843e33da930.png)
can you provide finetuning code? appreciated.

**Comment by xelda1988** [comment]
Hi. 
Thanks for your request.
I will have a look at it.

**Comment by xelda1988** [comment]
Hey @zdlpsina 

I could reproduce your error. It should work now:
Check it out:
https://github.com/deepopinion/domain-adapted-atsc/tree/master/finetuning_and_classification

I also checked the final files, they should be the same as in the models I released.

I will close this issue, let me know if everything works for you.
